interface/client/lib/helpers/helperFunctions.js

```diff
@@ -114,7 +114,7 @@ Helpers.generateBreadcrumb = function (url) {
         return el === '';
     });
 
-    return new Spacebars.SafeString(_.flatten(['<span>' + filteredUrl.host + ' </span>', pathname]).join(' ▸ '));
+    return new Spacebars.SafeString(url.protocol +'//'+ _.flatten(['<span>' + filteredUrl.host + ' </span>', pathname]).join(' ▸ '));
```

Comment:
@frozeman here I'd recommend using the Blaze._escape the same way we're using on `host` and `pathname`.